airbyte-integrations/connectors/source-mysql/src/main/kotlin/io/airbyte/integrations/source/mysql/MySqlSourceJdbcStreamStateValue.kt

```diff
@@ -40,7 +40,7 @@ data class MySqlSourceJdbcStreamStateValue(
                     Jsons.valueToTree(
                         MySqlSourceJdbcStreamStateValue(
                             cursorField = listOf(cursor.id),
-                            cursors = Jsons.writeValueAsString(cursorCheckpoint),
+                            cursors = cursorCheckpoint.asText(),
```

Comment:
This caused acceptance tests to fail for a reason that I couldn't figure out. 
I reverted the change the original. Still DecimalNodes are created in a safer way then before to minimize the chance of scientific notation.